helm-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "json-rpc-relay.fullname" . }}
+  labels:
+    app:  {{ template "json-rpc-relay.name" . }}
+    {{ include "json-rpc-relay.labels" . | nindent 4 }}
+data:
+{{- if .Values.config.local.HEDERA_NETWORK }}
+  HEDERA_NETWORK: {{ .Values.config.local.HEDERA_NETWORK | toJson }}
+{{- else if .Values.config.hosted.HEDERA_NETWORK }}
+  HEDERA_NETWORK: {{ .Values.config.hosted.HEDERA_NETWORK }}
+{{- else }}
+  HEDERA_NETWORK: {{ required "A valid HEDERA_NETWORK must be present in either .Values.config.local.HEDERA_NETWORK or .Values.config.hosted.HEDERA_NETWORK " (or .Values.config.local.HEDERA_NETWORK .Values.config.hosted.HEDERA_NETWORK) }}
+{{- end }}  
+  OPERATOR_ID_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_ID_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
+  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: {{ .Values.config.OPERATOR_KEY_ETH_SENDRAWTRANSACTION | default (printf "%q" "") }}
```

Comment:
Given the nature of the details shouldn't this key also be a secret?

Reply:
Moved these two items to secrets and out of configmaps